modules/clearComplete.js

```diff
@@ -0,0 +1,5 @@
+const clearCompletedTasks = (tasks) => {
+  const arr = tasks.filter((task) => !task.completed);
+  return arr;
+};
```

Comment:
This could have been re-written as;
`const clearCompletedTasks = (tasks) => tasks.filter((task) => !task.completed)`